src/sunburst/index.js

```diff
@@ -100,14 +101,15 @@ class Sunburst extends React.Component {
         <ArcSeries {...{
           colorType,
           ...this.props,
-          animation: false,
+          animation,
           radiusDomain: [0, radialDomain],
-          data: mappedData
+          // need to present a stripped down version for interpolation
+          data: animation ? mappedData.map(row => ({...row, parent: null, children: null})) : mappedData,
```

Comment:
this same technique will most likely need to be used if we move the treemap to use XYPlot as well...